t/full-stack.t

```diff
@@ -135,7 +135,7 @@ $driver->find_element_by_link_text('core@coolone')->click();
 $driver->title_is("openQA: $job_name test results", 'scheduled test page');
 my $job_page_url = $driver->get_current_url();
 like(status_text, qr/State: scheduled/, 'test 1 is scheduled');
-ok javascript_console_has_no_warnings_or_errors, 'no javascript warnings or errors after test 1 was scheduled';
+ok(javascript_console_has_no_warnings_or_errors(), 'no unexpected js warnings after test 1 was scheduled');
```

Comment:
are the outer parentheses needed? I prefer the style without

Reply:
The lack of parantheses were the reason the code was broken in an ealier version. And since I can't read it properly I'm leaving them

Reply:
What do you mean with "can't read it properly"?

Reply:
https://github.com/os-autoinst/openQA/pull/4288#discussion_r732920254